### ISSUE TITLE
Not Enough Memory When Adding

### ISSUE THREAD

**Issue by rushsteve1** [YunoHost-Apps/wallabag2_ynh#64]
When attempting to add an article using either Wallabagger or the Wallabag2 web interface it fails, and when using the web interface I get the following error:

```
500: Internal Server Error

Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 134217736 bytes)
```

Wallabag version: 2.3.2
Yunohost version: 3.5.2.2

**Comment by lapineige** [comment]
That was close :smile: (8 bytes)

Do you have the same behaviour with all articles ?

**Comment by rushsteve1** [comment]
Yep, doesn't matter the article they all won't save. You would think the 8 bytes wouldn't make a big difference...

Also I confirmed that trying to save with the mobile app doesn't work either.

**Comment by lapineige** [comment]
Do you have some articles already saved in wallabag, or is it a fresh install ?

I suppose it's an issue with php config… did you change anything regarding this ?

**Comment by rushsteve1** [comment]
I have things already saved. I was using it normally and then one day it stopped working. Didn't touch any PHP config or anything.

**Comment by lapineige** [comment]
Did you upgrade it or something like that ?

**Comment by rushsteve1** [comment]
I don't think I did but I might have. I don't quite remember. When was the last published release? I tend to keep up with updates so whenever that came out I probably got it.

**Comment by lapineige** [comment]
There were no recent update in master, so that's probably not the issue.

@JimboJoe do you have any clue ?

**Comment by dosch** [comment]
This issue is due to your recent PHP upgrade.

Wallabag released an update to solve this issue. I hope the Yunohost package can be updated as well soon.

**Comment by lapineige** [comment]
See #71 :slightly_smiling_face:

**Comment by lapineige** [comment]
Will be fixed once #70 is merged :slightly_smiling_face:
